pkg/manifest/os.go

```diff
@@ -732,6 +732,10 @@ func (p *OS) serialize() osbuild.Pipeline {
 		pipeline.AddStage(osbuild.NewOscapRemediationStage(p.OpenSCAPConfig))
 	}
 
+	for _, dropin := range p.SystemdUnit {
+		pipeline.AddStage(osbuild.NewSystemdUnitStage(dropin))
+	}
+
```

Comment:
I didn't catch this during the review.  I noticed that a lot of cloud images were being rebuilt when they shouldn't have been affected (https://gitlab.com/redhat/services/products/image-builder/ci/images/-/jobs/4890881075).
This is duplicating the systemd unit stage in the pipeline since it's already defined.  The stage isn't new.

Reply:
Fixed and amended.

Reply:
@thozza Can you take a quick look at this so I don't self-approve?

Reply:
done, but I don't like some parts...